src/NodeJS/Server.php

```diff
@@ -254,6 +262,52 @@ public function getConnection()
     }
 
     /**
+     * Return the options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->options;
+    }
+
+    /**
+     * Set options array.
+     */
+    public function setOptions($options)
+    {
+        // pass these to the setOption() method which will validate.
+        foreach ($options as $key => $value) {
+            $this->setOption($key, $value);
+        }
+    }
+
+    /**
+     * Set a single option.
+     */
+    public function setOption($option, $value)
+    {
+        $valid_options = array(
```

Comment:
Actually better not to validate any options at all, because:

* in future Zombie versions new options could be added and driver won't be able to set them unless we update it each time
* we don't validate `$value` and for example developer can pass string instead of boolean and Zombie would fail anyway